src/common/components/clusterConfiguration/OpenShiftVersionSelect.tsx

```diff
@@ -23,6 +30,18 @@ const getOpenshiftVersionHelperText = (versions: OpenshiftVersionOptionType[]) =
         &nbsp;Please note that this version is not production ready. <OpenShiftLifeCycleDatesLink />
       </>
     );
+  } else if (
+    isSelectedVersionInDataFile(selectedVersionValue) &&
```

Comment:
Why not just? Do we really need a function for doing this?
```suggestion
    selectedVersionValue in openshiftVersionData['versions'] &&
```

Reply:
You're right, we don't